Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonHelpCommand.cs

```diff
@@ -18,28 +18,29 @@ public class KryptonHelpCommand : KryptonCommand
     {
         #region Instance Fields
 
-        private ButtonSpecAny _helpButtonSpec;
+        private ButtonSpecAny? _helpButtonSpec;
 
         #endregion
 
         #region Public
 
-        public ButtonSpecAny HelpButton
+        /// <summary>Gets or sets the help button.</summary>
+        /// <value>The help button.</value>
+        [DefaultValue(null), Description(@"Access to the help button spec.")]
+        public ButtonSpecAny? HelpButton
```

Comment:
This needs the `[AllowNull]` attribute to tell the compiler that it can accept null's but will not return them.

Reply:
@Smurf-IV Is there an example, so I can properly implement it?

Reply:
Addressed feedback